timer/src/main/java/com/toggl/timer/log/ui/SwipeActionCallback.kt

```diff
@@ -30,10 +29,14 @@ class SwipeActionCallback(
     override fun getSwipeDirs(
         recyclerView: RecyclerView,
         viewHolder: RecyclerView.ViewHolder
-    ): Int = when (viewHolder) {
-        is TimeEntryLogViewHolder -> ItemTouchHelper.LEFT.or(ItemTouchHelper.RIGHT)
-        else -> ItemTouchHelper.ACTION_STATE_IDLE
-    }
+    ): Int =
+        when (viewHolder) {
+            is TimeEntryGroupViewHolder -> ItemTouchHelper.LEFT.or(ItemTouchHelper.RIGHT)
```

Comment:
It reads nicer to me in full infix style (here and elsewhere in this file)
```suggestion
            is TimeEntryGroupViewHolder -> ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT
```